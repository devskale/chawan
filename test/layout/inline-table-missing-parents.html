<!DOCTYPE html>
<div style="display: table-row">
<span>test 1
<div style="display: inline-table; line-height: 1em"><div style="display: table-row"><div style="display: table-cell">test 2</div></div></div>
<!--
<div style="display: inline-block"><div><div>test</div></div></div>
 -->
test 3<br>test 3.5</span>
</div>
<span>test 4 <div style="display: table-row; background: red">test 5</div> test 6</span>
<br>
<span>test 7 <div style="display: table-cell">test 8</div> test 9<br>test 9.5</span>
<div style="display: table-row; background: red">
<div>thing</div>
<div>thing2</div>
thing2.5
<span>thing3</span>
<div>thing4</div>
<div style="display: table-cell">
thing3
</div>
</div>
<div style="display: table-row"><div style="display: table-cell; background: blue">test</div></div>
