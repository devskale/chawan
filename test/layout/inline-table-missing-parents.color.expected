test 1 test 2 test 3
test 3.5
test 4 [48;2;255;0;0mtest 5[49m test 6
test 7 test 8 test 9
test 9.5
[48;2;255;0;0mthing          thing3[49m
[48;2;255;0;0mthing2               [49m
[48;2;255;0;0mthing2.5 thing3      [49m
[48;2;255;0;0mthing4               [49m
[48;2;0;0;255mtest           [49m
